wire-ios-data-model/WireDataModel.xcodeproj/project.pbxproj

```diff
@@ -5228,6 +5230,13 @@
 		};
 /* End XCConfigurationList section */
 
+/* Begin XCLocalSwiftPackageReference section */
+		019C2B722D5E86C800ED67B9 /* XCLocalSwiftPackageReference "../WireCoreCrypto" */ = {
```

Comment:
I think this needs to be removed.
In case the Swift package is part of the workspace, we must not additionally add it as dependency.
Otherwise it's linked more than once:
<img width="422" alt="Screenshot 2025-02-17 at 13 55 10" src="https://github.com/user-attachments/assets/643bc035-7d92-46ba-9aeb-1dfa0380c36b" />

Reply:
Please also remove the xcframework reference from WireDataModel.

Reply:
done, I remove the old xcframework reference and the WireCoreCrypto Package (aka Package in WireDataModel project's packages)